src/sentry/templatetags/sentry_react.py

```diff
@@ -92,10 +92,11 @@ def get_react_config(context):
         messages = []
         is_superuser = False
 
+    enabled_features = []
+
```

Comment:
Is moving this intentional? Doesn't seem to do anything, I'd just back this out.